k8s/perftest/common/divorce/div-cos.yaml

```diff
@@ -28,54 +28,16 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
       image: hmctspublic.azurecr.io/div/cos:prod-135ec6cc
```

Comment:
To do: update this git hash once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged